plugins/utils/mini.nix

```diff
@@ -1,27 +1,24 @@
 {
   lib,
-  helpers,
-  config,
   pkgs,
   ...
 }:
-with lib;
-let
-  cfg = config.plugins.mini;
-in
-{
-  options.plugins.mini = {
-    enable = mkEnableOption "mini.nvim";
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "mini";
+  originalName = "mini-nvim";
```

Comment:
I think original name is `mini.nvim`?

I assume this is also the only reason you had to build a bespoke package option below, too?